encrypted-media-respec.html

```diff
@@ -2816,7 +2816,7 @@
         <section>
           <h4><a>MediaEncryptedEventInit</a></h4>
           <div><pre class="idl">dictionary MediaEncryptedEventInit : EventInit {
-             DOMString    initDataType = "";
+             required DOMString    initDataType
```

Comment:
It seems a bit odd to require an `initDataType` if `initData` itself isn't required?